packages/browser-extension/src/_locales/en/messages.json

```diff
@@ -62,6 +62,9 @@
   "reportDialog_popupShowUpOption": {
     "message": "Popup showed up"
   },
+  "reportDialog_ScrollingIssues": {
+    "message": "Scrolling Issues"
```

Comment:
Please update the message to `Cannot scroll` and update the rest of the translations in consequence to this change.

Reply:
Done.

Reply:
done